eslint.config.mjs

```diff
@@ -0,0 +1,145 @@
+import importPlugin from 'eslint-plugin-import-x';
+import react from 'eslint-plugin-react';
+import globals from 'globals';
+import babelParser from '@babel/eslint-parser';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+import js from '@eslint/js';
+import { FlatCompat } from '@eslint/eslintrc';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all,
+});
+
+export default [
+  {
+    ignores: [
+      '**/.vscode/',
+      '.github/',
+      'media/',
+      '**/static/',
```

Comment:
Will that not exclude e.g. `pontoon/base/static`?

Reply:
Yes it would 🤦🏼